src/components/Map/PlantingSiteMap.tsx

```diff
@@ -104,15 +118,27 @@ export default function PlantingSiteMap(props: PlantingSiteMapProps): JSX.Elemen
   );
 
   // fetch polygons and boundaries
+  const [includedLayers, setIncludedLayers] = useState<MapLayer[] | undefined>(layerOptions);
   useEffect(() => {
     const fetchPlantingSite = () => {
       const site = extractPlantingSite(plantingSite);
       const zones = extractPlantingZones(plantingSite);
       const subzones = extractSubzones(plantingSite);
 
+      const includedSources = new Array<MapSource>();
+      if (includedLayers?.includes('Sub-Zones')) {
```

Comment:
ok i think the plants dashboard fix is to check if includedLayers === undefined.. in which case just push the sources

Reply:
That seems like a good fix 👍